plugin/trino-iceberg/pom.xml

```diff
@@ -569,6 +571,40 @@
             </build>
         </profile>
 
+        <profile>
+            <id>jdbc-tests</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <includes>
+                                <include>**/io/trino/plugin/iceberg/catalog/jdbc/Test*.java</include>
+                            </includes>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <profile>
+            <id>rest-tests</id>
```

Comment:
if you combine them into single profile, you can call it eg `additional-catalog-tests`